README.md

```diff
@@ -122,9 +124,21 @@ The releasinator enforces certain conventions.  If a filename closely matches th
 4. `CHANGELOG.md` - This file is the source of truth for the releasinator.  The file should be organized with the most recent release on top, and formatted correctly.  The latest release is the one used when the releasinator executes, so it is a precondition that the `CHANGELOG.md` has been edited and committed **prior to releasing**.
   1. Releases either are contained within an Alt-H2 (`------`) or `##H2` format.  Any other format will be rejected.
   2. Each release MUST start with the release version, and may contain any following text, such as the date, and/or any release summary.
-  3. Releases MUST not skip versions (such as `1.0.0 -> 1.0.2`, or `1.0.2 -> 1.1.1`).  That is, if you skip or botch a release, it MUST be documented as an official release with a minimal description suggesting that this release is not actually available, or was skipped. This is for the benefit future maintainers of the repo to know that certain releases were not actually published or are not to be used (for whatever reason). In addition, external developers benefit when they look to upgrade versions and want to know what changes were made since a particular release. They can rest at ease knowing that there are no unaccounted-for releases. 
+  3. Releases MUST not skip versions (such as `1.0.0 -> 1.0.2`, or `1.0.2 -> 1.1.1`).  That is, if you skip or botch a release, it MUST be documented as an official release with a minimal description suggesting that this release is not actually available, or was skipped. This is for the benefit future maintainers of the repo to know that certain releases were not actually published or are not to be used (for whatever reason). In addition, external developers benefit when they look to upgrade versions and want to know what changes were made since a particular release. They can rest at ease knowing that there are no unaccounted-for releases.
 5. `.gitignore` and `.DS_store`: While this file is Mac-specific, many repos contain this entry in their `.gitignore` files because it is quite common for developers to have their global `.gitignore` configured incorrectly. Therefore, the authors of this project have made the decision to force this entry in all `.gitignore` files as a gesture of goodwill to all these new git users.
 
+## Auto-updating of version and CHANGELOG
+
+For your first release, the releasinator expects that the CHANGELOG contains an initial entry and that the initial version is placed as appropriate in relevant files.
+
+For subsequent releases, the releasinator handles updating the CHANGELOG and version numbers automatically, as part of the release flow. Specifically, it:
```

Comment:
Currently, it is mandatory. We could make it optional, maybe by allowing users to respond "skip" during the major/minor/patch prompt? The language of the prompt could be accordingly reconfigured to make it clear that this is an optional step for updating the version and changelog.

Or, if this method is present, use it - otherwise don't?

I agree that it makes sense as optional, for example for releases where the CHANGELOG entry is significant and merits out-of-flow work.